nifi-registry/nifi-registry-core/nifi-registry-docker/dockerhub/README.md

```diff
@@ -124,6 +124,39 @@ The following, optional environment variables may be added to the above command
     -e LDAP_TLS_TRUSTSTORE_PASSWORD: ''
     -e LDAP_TLS_TRUSTSTORE_TYPE: ''
 
+### Standalone Instance secured with HTTPS and OpenID Authentication
+In this configuration, the user will need to provide certificates and associated configuration information. 
+Of particular note, is the `AUTH` environment variable which is set to `oidc`. Additionally, the user must provide a
+in the `INITIAL_ADMIN_IDENTITY` environment variable. This value will be used to seed the instance with an initial 
+user with administrative privileges.
+
+### For a minimal, connection to an OpenID server
+
+    docker run --name nifi-registry \
+      -v $(pwd)/certs/localhost:/opt/certs \
+      -p 18443:18443 \
+      -e AUTH=oidc \
+      -e KEYSTORE_PATH=/opt/certs/keystore.jks \
+      -e KEYSTORE_TYPE=JKS \
```

Comment:
PKCS12 is the preferred format and the default starting from Java 9 on, so this example should be adjusted.
```suggestion
      -e KEYSTORE_PATH=/opt/certs/keystore.p12 \
      -e KEYSTORE_TYPE=PKCS12 \
```